test.js

```diff
@@ -110,7 +147,7 @@ function compareStdout(done, sourceMap, source, expected) {
       compareLines(
         (stdout + stderr)
           .trim()
-          .split('\n')
+          .split(/\r\n|\r|\n/g)
```

Comment:
Standalone `\r` is only used on Mac OS classic which no one uses today. How about `/\r?\n/g`?